judo-tatami-jsl-jsl2ui/src/main/epsilon/transformations/ui/modules/view/rowDeclaration.etl

```diff
@@ -173,6 +173,7 @@ rule TableTable
         t.name = s.name + "::Table";
         t.dataElement = s.map.transfer.equivalent("ClassType");
         t.relationName = "";
+        t.isInlineEditable = true;
```

Comment:
shouldn't this be a value coming from the source model?

Reply:
For JSL it will be on by default.